addons/test_mail_full/tests/test_sms_composer.py

```diff
@@ -145,9 +146,22 @@ def test_composer_default_recipient(self):
                     'number_field_name': 'phone_nbr',
                 })
 
-        self.assertFalse(composer.recipient_single_valid)
+        self.assertTrue(composer.recipient_single_valid)
```

Comment:
Hey, indeed if there's no partner to fallback on it doesn't make sense ^^'

Maybe having the commit will help a bit here https://github.com/odoo-dev/odoo/commit/02067a3342b3e187f0de0a900c4c5b6ed678808a

The idea is simply that if the recipient is invalid and we can't find info from `_sms_get_recipients_info`, then we should fallback on the default recipient for the record. So here if recipient_single_valid was not empty, it meant we did find something in  `_sms_get_recipients_info`, so we were not testing the same flow.

But since now we're doing the fallback inside `sms_get_recipients_info`, I guess the line can be removed actually.
```py
or records._sms_get_default_partners().display_name
```

In that case you can remove this test and just check
```
self.assertEqual(composer.recipient_single_description, self.test_record.customer_id.display_name)
```
in your test

Reply:
Oh I see, thank you for help, I removed the check :ok_hand: